CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 #### General
 * [Enhancement] Updated vendored version of boto3 to 1.12.6
 * [Enhancement] Updated vendored version of botocore to 1.15.6
+* [Enhancement] Add urllib3 to vended packages
```

Comment:
Could we add the version to this line? Looks like 1.25.9 judging from _version.py.

Reply:
Sure can

Reply:
LICENSE added to urllib3 directory, changelog updated to include version number.